docs/api/target.rst

```diff
@@ -28,6 +30,14 @@ COMMAND
 NUM_MPI_TASKS
   Indicates this is an MPI test and how many MPI tasks to use.
 
+NUM_OMP_THREADS
+  Indicates this test requires the defined environment variable ``OMP_NUM_THREADS``
+  set to the given variable.
+
+CONFIGURATIONS
+  Set the CTest configuration for this test.  When not specified, the test
+  will be added to BLT's default ``Benchmark`` CTest configuration.
```

Comment:
The code is:
```cmake
CONFIGURATIONS Benchmark ${arg_CONFIGURATIONS})
```
If I'm understanding it correctly, the docs are a bit misleading since it is always adding the test to the `Benchmark` configuration.

Perhaps
```suggestion
  Optionally set additional CTest configuration(s) for this test. Benchmark tests
  will always be added to the ``Benchmark`` CTest configuration.
```

Reply:
I'll fix this in the other PR after I have fixed the CI due to the Toss4 conversion.